docs/metasploit-framework.wiki/Setting-Up-A-AD-CS-Target.md

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Also shouldn't the file name be `...An-AD-CS...` instead of `...A-AD-CS...`?

```suggestion
```

Reply:
Good point will fix this up now

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17399/commits/239bc42f4c4228a21dd726e08fa3273545112f7f